packages/adminrouter/extra/src/test-harness/tests/test_generic.py

```diff
@@ -0,0 +1,478 @@
+# Copyright (C) Mesosphere, Inc. See LICENSE file for details.
```

Comment:
I was worried more about YAML verification, but I've noticed a comment that you're planning to use JSON schema validator.